tests/unit/test_cohere_encoder.py

```diff
@@ -0,0 +1,97 @@
+import pytest
```

Comment:
Can you please add a system test similar to the one we have for openai? (it's under `tests/system`)

Reply:
I already put a secret on github under  `CO_API_KEY` env var, so the test can simply pass `None` to the encoder in the api key